deepchecks/tabular/checks/model_evaluation/simple_model_comparison.py

```diff
@@ -31,18 +32,31 @@
 
 __all__ = ['SimpleModelComparison']
 
+_allowed_strategies = (
+    'most_frequent',
+    'uniform',
+    'tree'
+)
+
+_depr_strategies = {
+    'random': 'uniform',
+    'constant': 'most_frequent',
+}
+
 
 class SimpleModelComparison(TrainTestCheck):
     """Compare given model score to simple model score (according to given model type).
 
     Parameters
     ----------
-    simple_model_type : str , default: constant
-        Type of the simple model ['random', 'constant', 'tree'].
-        + random - select one of the labels by random.
-        + constant - in regression is mean value, in classification the most common value.
-        + tree - runs a simple decision tree.
-    alternative_scorers : Dict[str, Callable], default None
+    strategy : str, default: 'constant'
+        Strategy to use to generate the predictions of the simple model ['uniform', 'most_frequent', 'tree'].
+        * `uniform`: select one of the labels by random. (Previously 'random')
```

Comment:
If i understood correctly, this is NOT random. it selects a value at random based on the labels distribution.
Should be a different name.
In addition would suggest adding a true random - selects at random from labels.unique_values()

Reply:
think it should be done now to avoid additional api changes in future
Same in vision

Reply:
Hmmm, I was misled by the docs. You're right